handout-files/README.md

```diff
@@ -266,10 +266,13 @@ overriding those methods (and calling the `super` method of course).
 Included in this handout is a tool for visualizing and graphically debugging
 executions of distributed systems. You can run it in two different ways. First,
 you can start any lab from its initial state by calling `./run-tests.py --lab N
---debug NUM_SERVERS NUM_CLIENTS WORKLOAD`, where `WORKLOAD` is a comma-separated
-list of commands for the clients to send (e.g.,
-`PUT:foo:bar,APPEND:foo:baz,GET:foo`). You can even write custom parsers for the
-arguments to `--debug` by overriding `getInitialState(String[] args)` in the
+--debug NUM_SERVERS NUM_CLIENTS WORKLOAD` or `./run-tests.py --lab N --debug
+NUM_SERVERS NUM_CLIENTS WORKLOAD_1 ... WORKLOAD_N`. Here `WORKLOAD`/`WORKLOAD_i`
```

Comment:
`WORKLOAD_NUM_CLIENTS`. `N` conflicts with `--lab N` anyways

Reply:
Fixed in [776c63a](https://github.com/emichael/dslabs/pull/19/commits/776c63ab12e92c0fa40dd1354bdb20f736cc805e)